packages/framework-core/src/services/read-model-store.ts

```diff
@@ -1,87 +1,66 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {
   BoosterConfig,
-  ReadModelInterface,
-  ProjectionMetadata,
-  UUID,
+  BoosterMetadata,
   EntityInterface,
-  ReadModelAction,
+  EntityMetadata,
+  EntitySnapshotEnvelope,
+  FilterFor,
   OptimisticConcurrencyUnexpectedVersionError,
-  SequenceKey,
   ProjectionGlobalError,
-  EntitySnapshotEnvelope,
-  BoosterMetadata,
+  ProjectionMetadata,
+  ProjectionResult,
+  ReadModelAction,
+  ReadModelInterface,
+  ReadModelJoinKeyFunction,
+  SequenceKey,
+  UUID,
 } from '@boostercloud/framework-types'
-import { Promises, retryIfError, createInstance, getLogger } from '@boostercloud/framework-common-helpers'
-import { BoosterGlobalErrorDispatcher } from '../booster-global-error-dispatcher'
+import { createInstance, getLogger, Promises, retryIfError } from '@boostercloud/framework-common-helpers'
+import { readModelSearcher } from './read-model-searcher'
 import { ReadModelSchemaMigrator } from '../read-model-schema-migrator'
+import { BoosterGlobalErrorDispatcher } from '../booster-global-error-dispatcher'
 
 export class ReadModelStore {
   public constructor(readonly config: BoosterConfig) {}
 
   public async project(entitySnapshotEnvelope: EntitySnapshotEnvelope): Promise<void> {
-    const logger = getLogger(this.config, 'ReadModelStore#project')
+    const logger = getLogger(this.config, 'ReadModelStore#projectEntities')
```

Comment:
I guess we can leave the logger as it was, since `project` function has not been renamed: `const logger = getLogger(this.config, 'ReadModelStore#project')`